tests/core/pyspec/eth2spec/test/altair/merkle/test_single_proof.py

```diff
@@ -8,8 +8,14 @@
 
 @with_phases([ALTAIR])
 @spec_state_test
-def test_next_sync_committee_tree(spec, state):
+def test_next_sync_committee_merkle_proof(spec, state):
+    yield "state", state
     next_sync_committee_branch = build_proof(state.get_backing(), spec.NEXT_SYNC_COMMITTEE_INDEX)
+    yield "proof", {
+        "leaf": "0x" + state.next_sync_committee.hash_tree_root().hex(),
+        "leaf_index": spec.NEXT_SYNC_COMMITTEE_INDEX,
+        "branch": ['0x' + root.hex() for root in next_sync_committee_branch]
```

Comment:
Is `root` the common name even for intermediate hashes?

Reply:
In remerkleable every 32-byte tree thing is a "root", (`Root` type). And the tree is built with `PairNode` and `RootNode` types. If you take the merkle-root of any subtree (even if just a single node) it's a `Root`. It's a little confusing with the "leaf/sibling" meanings, but it's the opposite of a pair, so I stuck with it. Naming is hard.